modules/claims_api/app/controllers/concerns/claims_api/mpi_verification.rb

```diff
@@ -5,25 +5,8 @@ module MPIVerification
     extend ActiveSupport::Concern
 
     included do
-      before_action :verify_mpi
```

Comment:
I like this.  It seemed odd to me that the concern would be declaring a `before_action` to basically run itself.  It makes more sense (to me) to allow the controller that's pulling this in to decide when the logic should be ran.

`ClaimsApi::PoaVerification` concerns has that same odd pattern.

Reply:
youre right! made the same change to PoaVerification
https://github.com/department-of-veterans-affairs/vets-api/pull/6333/commits/6268d295eb63ff60dc7c90fd6dfb44cd73d03ee5